db/migrate/20190617103359_add_success_more_than_fifty_percent_to_merits_assessments.rb

```diff
@@ -0,0 +1,5 @@
+class AddSuccessMoreThanFiftyPercentToMeritsAssessments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :merits_assessments, :success_more_than_fifty_percent, :boolean
```

Comment:
What happens if the `likely` threshold is changed? If it was changed to 60% would this column need to be renamed? Would `success_likely` be a better name?

Reply:
yes much better!

Reply:
I changed everything to `success_likely`
attribute, route, controller, form.

The route is now `resources :success_likely, only: %i[index create]`

I'm using `resources` with `%i[index create]` so the controller doesn't have to be plural.

`likelies`!? :scream:

Reply:
There's always `likely_success` which would have the `likely_successes` plural if needed perhaps.

Reply:
I think it's fine like that. We also use `resources` and `%i[index create]` for other pages.